### DISCUSSION TITLE
Migration Regarding V3 to V5

### DISCUSSION THREAD

**Discussion by TamilB** [remix-run/react-router#8170] [Q&A]
Hi ,
Any one Please Help me to migrate react-router version "3.0.5" to latest one

**Comment by mjackson** [comment]
It's probably easier to migrate from v3 to v6 now that v6 is released. The routing structure in v6 is much more similar to v3 than v5.

**Reply by cseas** [reply]
Is there a guide available on the v3 to v6 upgrade? The people at my workplace are stuck at v3 (onEnter) and were waiting for v6 but there are no guides available on how to do the upgrade.
